contents/handbook/company/communication.md

```diff
@@ -208,14 +208,19 @@ Here are the steps for an RFC:
 4. If an RFC is cross-team and is causing a large amount of disagreement, it might be worth having a sync meeting to reach a decision
 5. Once a decision is made, the person making the decision should include the decision in the pull request, merge it in and share this in the relevant channel and `#github-rfcs` again.
 
-### When should you write an RFC?
+### When does it work best to write an RFC?
 
-You most likely **should** write an RFC when any of the following is true (this is a non-exhaustive list):
-- It affects multiple teams
+Writing an RFC may be helpful when any of the following is true:
+- You want to clarify something for yourself or it affects just one team
+- It is a relatively non contraversial - for example, not creating problems for another team - change/idea without much extra context needed
 - It will be a large amount of work (more than 2-3 weeks of people's time)
 - It's introducing a new technology
 - It's a major new feature, change to the product, or change to the company
-- It will have a major customer impact, e.g. pricing
+- It will have a major customer impact
+
+### When does a meeting / another approach work better than an RFC?
+
+An RFC is likely to be unhelpful in other circumstances. Specifically, when you want to ship or suggest a change to something that significantly affects teams outside your own. In this instance, we've seen that RFCS can lead to 10 to 25+ comments, which feels antagonistic (teams having to explain all the context around their strategy down to why this decision is something they perhaps disagree with), and creates a lot of work. A single call in this instance is likely much faster than lots of frustrated people in 1/1s talking about it _and_ the energy/time needed to respond to everything in a long thread.
```

Comment:
I'm a little dubious about this because sometimes in a meeting people can verbally agree to things, but when it's fully written down and spelled out they realize that there are gaps. And writing things down fully can be really helpful for fully thinking through things. I wonder if it would be a good practice to follow up a meeting with an RFC to make sure all teams are on the same page.

Reply:
Agree the addition of suggesting to follow up as an RFC or something makes a lot of sense

Reply:
yeah that feels better - it's having a bit of a chat first _then_ means an RFC is just a clarification exercise as opposed to I AM RIGHT YOU ARE WRONG!

Reply:
Agree that a call followed by an clarifying RFC makes sense. That said, I think you still need something before a meeting so people have time to consider what they're about to talk about, like an RFC-lite.

Reply:
i've updated to reflect this!